### DISCUSSION TITLE
The should('not.be.visible') failed when html element get covered by fixed full page dialog

### DISCUSSION THREAD

**Discussion by ghost** [cypress-io/cypress#26463] [Questions and Help]
Hello,

How does Cypress count visible?
If the idea of "scroll" is involved, the should('not.be.visible') works as expected.
But in situations with html element get covered by fixed full page dialog, it failed.

Thanks

**Comment by michallbanas** [comment]
Hey @yaobao1993 👋 

you can find informations about visibility in [Cypress documentation](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Visibility). 

Can you provide more informations for your specific case? 🙏(eg. screenshot of failed tests, error message etc.)
